serialization/serialization_writer_proxy_factory.go

```diff
@@ -0,0 +1,86 @@
+package serialization
+
+type ParsableAction func(Parsable)
```

Comment:
doc comments?
we probably also need to allow them to return an error

Reply:
Done